packages/platform-sdk-profiles/src/drivers/memory/profiles/profile.initialiser.ts

```diff
@@ -44,7 +49,10 @@ export class ProfileInitialiser implements IProfileInitialiser {
 		this.#profile.settings().set(ProfileSetting.UseCustomPeer, false);
 		this.#profile.settings().set(ProfileSetting.UseMultiPeerBroadcast, false);
 		this.#profile.settings().set(ProfileSetting.UseTestNetworks, false);
+	}
 
-		emitProfileChanged();
+	/** {@inheritDoc IProfileInitialiser.initialiseData} */
+	public initialiseData(): void {
+		this.#profile.data().set(ProfileData.HasCompletedTutorial, false);
```

Comment:
Instead of initialising a default value you should introduce a `profile.hasCompletedTutorial()` method which checks for `return this.#profile.data().get(ProfileData.HasCompletedTutorial) === true;`

Reply:
so you mean after Completed Tutorial by user then i set a data?

Reply:
nice, good idea 
i'll implement it